### PR TITLE
Discover host ip address netmasks

### PR DESCRIPTION
**Pull request by arbulu89**
# Description

Discover host netmasks and send them in the` netmasks` field. Each value corresponds the value in the same position for `ip_addresses`.

I'm aware that this "position related" usage is not optimal, and actually, in the 1st commit I implemented a more complex struct to send complete network interfaces. But this makes compatibility between old web/new agent, new web/old agent more difficult, so I sticked to simplest solution.

If for some reason, in the future we need more information about network interfaces, I finally think that we will need something more complex implementation that eventually will require some upgrade in api versioning or even major version of agent/web.

## How was this tested?

Partially tested...
We don't have any code testing host discoveries, and in fact, testing `network` related things is pretty  difficult.
Basically the serialization as json is the unique thing that it is tested.
In any case, the used functions are not changed, just the composed final code, so as everything was working now, i expect that it will continue working

### CODE REVIEW

File: test/fixtures/discovery/host/expected_published_host_discovery.json [hunk 1]
```diff
@@ -1,19 +1,24 @@
...
+  "discovery_type": "host_discovery",
+  "payload": {
+    "os_version": "15-SP2",
+    "ip_addresses": [
+      "10.1.1.4",
+      "10.1.1.5",
+      "10.1.1.6"
+    ],
+    "netmasks": [
+      24,
```
Comment (EMaksy) [inline test/fixtures/discovery/host/expected_published_host_discovery.json:12]: The only thing what concerns me is we must just think about using always the correct index in trento web :)

**Review by EMaksy — APPROVED** [review]
LGTM!

File: test/fixtures/discovery/host/expected_published_host_discovery.json (hunk 1 above)
Comment (arbulu89) [reply test/fixtures/discovery/host/expected_published_host_discovery.json:12]: As the length of ip addresses and netmasks is always the same, you can use the standard `zip` and `unzip` functions to combine them pretty easily

Comment (EMaksy) [reply test/fixtures/discovery/host/expected_published_host_discovery.json:12]: Oh, today I learned about lodash [zipWith](https://lodash.com/docs/4.17.15#zipWith). Okay, this is cool!
Then never mind.

**Review by CDimonaco — APPROVED** [review]
